desktop/src/main/java/bisq/desktop/util/validation/PhoneNumberValidator.java

```diff
@@ -95,6 +105,8 @@ public ValidationResult validate(String input) {
             } else {
                 normalizedPhoneNumber = "+" + getCallingCode() + pureNumber;
             }
+
+            result = validateRequiredLength(normalizedPhoneNumber, normalizedCallingCode);
```

Comment:
I think the check if the requiredLength is 0 (or better ifPresent when optional is used) should be done here, as we only validate if that is the case.